conformance/runner/run_cv_bundle.py

```
@@ -2019,6 +2019,147 @@ def run_reorg(gate: str, fixture: dict[str, Any], failures: list[str]) -> int:
+        # PASS: require deterministic JSON output.
+        if p_r.returncode != 0:
+            failures.append(f"{gate}:{test_id}: rust expected PASS, got={tok_r} ({p_r.stderr.strip()})")
+            continue
+        if p_g.returncode != 0:
+            failures.append(f"{gate}:{test_id}: go expected PASS, got={tok_g} ({p_g.stderr.strip()})")
+            continue
+
+        try:
+            out_r = json.loads(p_r.stdout.strip())
+            out_g = json.loads(p_g.stdout.strip())
+        except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)